### ISSUE TITLE
backup from the subscribers added to pyhss

### ISSUE THREAD

**Issue by b3c3ll** [nickvsnetworking/pyhss#192]
Hi, 
Is there a way to backup informations that added to pyhss?
Each time i install pyhss and needs to add a subscriber, i need to do all the steps manually using the web interface! is there any way to backup and then upload the file to automatically add those users?

or any automated way instead of some hard manually adding subs to the program?

Thanks
Best regards

**Comment by laf0rge** [comment]
Isn't all the data stored in the SQL database that you configured to use (mysql, postgresql, sqlite, ...)?  Please look at backup instructions of the respective database system, like pg_dump, mysqldump, ...

**Comment by osmith42** [comment]
Yes, please use the database features for exportang and importing backups. With current master, sqlite has been added and is the default now, which means you can just stop PyHSS and copy the sqlite database file. Closing.
